src/pybind/mgr/cephadm/schedule.py

```diff
@@ -50,14 +50,14 @@ def place(self, host_pool, count=None):
 
 
 class HostAssignment(object):
-
-    def __init__(self,
-                 spec,  # type: ServiceSpec
-                 get_hosts_func,  # type: Callable
-                 get_daemons_func, # type: Callable[[str],List[orchestrator.DaemonDescription]]
-                 filter_new_host=None, # type: Optional[Callable[[str],bool]]
-                 scheduler=None,  # type: Optional[BaseScheduler]
-                 ):
+    def __init__(
+            self,
+            spec,  # type: ServiceSpec
+            get_hosts_func,  # type: Callable
+            get_daemons_func,  # type: Callable[[str],List[orchestrator.DaemonDescription]]
+            filter_new_host=None,  # type: Optional[Callable[[str],bool]]
+            scheduler=None,  # type: Optional[BaseScheduler]
+    ):
```

Comment:
```suggestion
    def __init__(
            self,
            spec: ServiceSpec,
            get_hosts_func: Callable,
            get_daemons_func: Callable[[str],List[orchestrator.DaemonDescription]],
            filter_new_host: Optional[Callable[[str],bool]] = None,
            scheduler: Optional[BaseScheduler] = None,
    ):
```

might as well covert these to py3 style too?

Reply:
Please do that in a separate PR.